elyra/pipeline/resources/properties.json

```diff
@@ -0,0 +1,372 @@
+        },
+        "description": {
+          "default": "Container image used as execution environment"
+        }
+      },
+      {
+        "parameter_ref": "cpu",
+        "label": {
+          "default": "CPU"
+        },
+        "description": {
+          "default": "For CPU-intensive workloads, you can choose more than 1 CPU (e.g. 1.5)."
+        }
+      },
+      {
+        "parameter_ref": "gpu",
+        "label": {
+          "default": "GPU"
+        },
+        "description": {
+          "default": "For GPU-intensive workloads, you can choose more than 1 GPU. Must be an integer."
+        }
+      },
+      {
+        "parameter_ref": "memory",
+        "label": {
+          "default": "RAM(GB)"
+        },
+        "description": {
+          "default": "The total amount of RAM specified."
+        },
+        "orientation": "horizontal",
+        "separator": "after"
+      },
+      {
+        "parameter_ref": "dependencies",
+        "label": {
+          "default": "File Dependencies"
+        },
+        "control": "custom",
+        "custom_control_id": "elyra-string-array-input",
+        "description": {
+          "default": "Local file dependencies that need to be copied to remote execution environment.\nOne filename or expression (e.g. *.py) per line. Supported patterns: ? and *."
+        },
+        "orientation": "horizontal",
+        "separator": "after",
+        "data": {
+          "single_item_label": "Dependency",
+          "placeholder": "*.py",
+          "filebrowser": true
+        }
+      },
+      {
+        "parameter_ref": "include_subdirectories",
+        "label": {
+          "default": "Include Subdirectories in Dependencies"
+        },
+        "description": {
+          "default": "May increase submission time"
+        },
+        "orientation": "horizontal",
+        "separator": "after"
+      },
+      {
+        "parameter_ref": "env_vars",
+        "label": {
+          "default": "Environment Variables"
+        },
+        "control": "custom",
+        "custom_control_id": "elyra-string-array-input",
+        "description": {
+          "default": "Environment variables to be set on the execution environment.\nOne variable per line in the format ENV_VAR=value."
+        },
+        "orientation": "horizontal",
+        "separator": "after",
+        "data": {
+          "placeholder": "env_var=VALUE",
+          "single_item_label": "Environment Variable"
+        }
+      },
+      {
+        "parameter_ref": "outputs",
+        "label": {
+          "default": "Output Files"
+        },
+        "control": "custom",
+        "custom_control_id": "elyra-string-array-input",
+        "description": {
+          "default": "Files generated during execution that will become available to all subsequent pipeline steps.\n One filename or expression (e.g. *.csv) per line. Supported patterns: ? and *."
+        },
+        "data": {
+          "placeholder": "foo.csv",
```

Comment:
This doesn't look right.  Perhaps a name like `outputs.csv` or `results.csv` would be more appropriate?